src/lib/layouts/workflow-header.svelte

```diff
@@ -122,6 +136,27 @@
           class="overflow-hidden text-ellipsis"
         />
       </h1>
+      <div class="flex gap-4 items-center">
+        <WorkflowStatus status={workflow?.status} />
+        {#if workflowUsesVersioning}
+          <p class="flex gap-1 items-center">
+            <span>{translate('workers', 'last-used-version')}</span
+            ><CompatibilityBadge
+              defaultVersion={buildId === overallDefaultVersion}
+              active={buildId === overallDefaultVersion}
+              {buildId}
+            />
+          </p>
+          <p class="flex gap-1 items-center">
+            <span>{translate('workers', 'next-version')}</span
+            ><CompatibilityBadge
```

Comment:
It there a default "none" we can show for the `CompatibilityBadge`? Or maybe a check for `Last used version` and `Next version` text so that there isn't text if there isn't a version?

![Screenshot 2023-07-28 at 3 18 59 PM](https://github.com/temporalio/ui/assets/15069288/ef65853a-6ccb-4e78-bbbd-a6ab8ad2cd03)

Reply:
How were you able to get this to happen? There should always be a default for a version set.

Reply:
Hmm maybe this is just a weird state a user won't see. I'm getting it because of the `501` "Not Implemented" error for the compatibility and worker-task-reachability apis.